lib/smart_proxy_pulp_plugin/pulp_node_plugin.rb

```diff
@@ -1,7 +1,8 @@
 module PulpMasterProxy
   class Plugin < ::Proxy::Plugin
     plugin "pulpnode", ::PulpProxy::VERSION
-    default_settings :pulp_url => 'https://localhost/pulp'
+    default_settings :pulp_url => 'https://localhost/pulp',
+                     :pulp_dir => '/var/lib/pulp'
```

Comment:
> yeah, monitoring the mongo path would be very helpful.  
> 
> It may also be useful to monitor /var/lib/pulp/content/ by itself, as sometimes that is on a different file system.

Instead of :pulp_dir this could be

    :monitor_dirs:
        - /var/lib/pulp
        - /var/lib/mongod

etc...

-- 
Later,
 Lukas #lzap Zapletal